themes/default/layouts/partials/banner.html

```diff
@@ -1,6 +1,6 @@
-{{ $summary := "Recently announced: Pulumi Business Critical Edition for Enterprise Modernization." }}
-{{ $url := relref . "/blog/business-critical-launch" }}
-{{ $label := "Read more"}}
+{{ $summary := "Join us virtually on May 4th as we introduce the last innovations in Cloud Engineering." }}
```

Comment:
is this meant to say "latest innovations in Cloud Engineering" instead of "last"

Reply:
```suggestion
{{ $summary := "Join us virtually on May 4th as we introduce the latest innovations in Cloud Engineering." }}
```

Yes good catch!